### PR TITLE
for issue 31

### PR DESCRIPTION
**Pull request by DJ-Eckam**

### CODE REVIEW

File: themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/_breadcrumbs.html [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{% if 0 == 1 %}
```
Comment (chrisflesher) [inline themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/_breadcrumbs.html:1]: The liquid has a `{% comment %}` and `{% endcomment %}` tags that do something similar and seem to be what's used commonly.

Comment (chrisflesher) [reply themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/_breadcrumbs.html:1]: https://nationbuilder.com/liquid_basics

File: themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/_nb_updates.scss [hunk 2]
```diff
@@ -22,9 +22,9 @@
   .main.width-container {
     background: transparent;
   }
-  .footer-nav {
-    display: none;
-  }
+  // .footer-nav {
+  //   display: none;
+  // }
```
Comment (chrisflesher) [inline themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/_nb_updates.scss:27]: What is removing these `display: none` for?

**Review by chrisflesher — APPROVED** [review]
Looks pretty good, just some minor suggstions / questions.

**Comment by chrisflesher** [comment]
After you're done addressing questions, feel free to merge. On the green button where it says "Merge pull request" please use the arrow to the right to change it to "Squash and merge" then edit the commit text to summarize the changes better.

File: themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/_nb_updates.scss (hunk 2 above)
Comment (DJ-Eckam) [reply themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/_nb_updates.scss]: it's unnecessary b/c that file isn't used, so I reverted

**Pull request merged**